resources/rules/post_identifier_merge/temp/abstraction/gene_or_gene_product_abstraction.clj

```diff
@@ -0,0 +1,20 @@
+;; ----------------------------------------------------
+;; --------- Gene or Gene Product Abstraction ---------
+;; ----------------------------------------------------
+;;TODO the restrictions need to be properly reified
+
+`{:name "gene-or-gene-product-abstraction"
+  :description "This rule creates an absraction for referring to genes or gene products"
+  ;; this needs to be hashed and have proper classes
+  :head ((?/gene_product rdfs/subClassOf ?/gene_or_gene_product))
+  
+  :body ((?/gene rdf/type kbio/GeneSpecificGClass)
+         (?/gene rdfs/subClassOf ?/gene_or_gene_product)
+         (?/gene_or_gene_product rdf/type kbio/GeneSpecificGorGPClass)
+
+         (?/has_indriect_template_restriction owl/someValuesFrom ?/gene)
+         (?/has_indriect_template_restriction owl/onProperty kso/has_indirect_template)
```

Comment:
@bill-baumgartner I support using something that already exists as long as it makes sense. I think that's also more in line with the spirit of what we are doing and will be easier to defend in publications. I will make the change.